src/routes/knowledge/+layout.svelte

```diff
@@ -13,11 +14,14 @@
 	<svelte:fragment slot="list-items">
 		{#if $knowledgeStore && $knowledgeStore.length > 0}
 			{#each $knowledgeStore as knowledge}
+				{@const subtitles = [
+					formatDistanceToNow(new Date(knowledge.updatedAt), { addSuffix: true })
+				]}
 				<SectionListItem
 					sitemap={Sitemap.KNOWLEDGE}
 					id={knowledge.id}
 					title={knowledge.name}
-					subtitle={knowledge.updatedAt}
+					subtitle={subtitles.join(' • ')}
 				/>
 			{/each}
```

Comment:
In general I try to avoid having _single-use abstractions_, since the `subtitle` it's only going to be a timestamp for now we can just pass it directly and avoid setting a `{@const subtitles}`:

```suggestion
			{#each $knowledgeStore as knowledge}
				<SectionListItem
					sitemap={Sitemap.KNOWLEDGE}
					id={knowledge.id}
					title={knowledge.name}
					subtitle={formatDistanceToNow(new Date(knowledge.updatedAt), { addSuffix: true })}
				/>
			{/each}
```